CMakeLists.txt

```diff
@@ -119,33 +130,23 @@ if (MSVC)
     set(OPENSSL_USE_STATIC_LIBS true)
     set(OPENSSL_MSVC_STATIC_RT true)
 
-    if (ZEEK_STANDALONE)
-        include(${CMAKE_SOURCE_DIR}/cmake/conan.cmake)
-        conan_cmake_autodetect(settings)
-        # Install packages from conanfile
-        conan_cmake_install(PATH_OR_REFERENCE ${CMAKE_SOURCE_DIR}/ci/windows/conanfile_windows.txt
-                            BUILD missing SETTINGS ${settings})
-    endif ()
-
-    # Set LibPCAP to point to libpcap binaries.
+    # Set PCAP_ROOT_DIR to point at the installation from vcpkg. A later call
+    # to FindPCAP.cmake will fill in the rest of the necessary variables.
     if (NOT PCAP_ROOT_DIR)
-        find_package(libpcap)
-        set(PCAP_ROOT_DIR "${libpcap_LIB_DIRS}/../")
-        set(PCAP_INCLUDE_DIR ${libpcap_INCLUDES})
-        set(PCAP_LIBRARY ${libpcap_LIBS})
+        set(PCAP_ROOT_DIR ${VCPKG_INSTALLED_DIR}/x64-windows-static)
     endif ()
 
     set(LIBPCAP_PCAP_COMPILE_NOPCAP_HAS_ERROR_PARAMETER false)
 
-    # Set ZLib to point at the right variable.
+    # Find zlib installed by vcpkg.
     find_package(ZLIB)
-    set(ZLIB_LIBRARY ${ZLIB_LIBRARIES})
+    set(ZLIB_LIBRARY ZLIB::ZLIB)
```

Comment:
Interesting, I like how tidy this looks. Let me make sure I get the workflow: it's the inclusion of the toolchain file for vcpkg that kicks this all off; this then pulls in vcpkg.json to install c-ares, libpcap, and zlib, and it populates `ZLIB::ZLIB` and `c-ares::cares` with the results. Is that roughly right?

Reply:
Correct. Vcpkg's CMake integration automatically creates the `ZLIB::ZLIB` and `c-ares::cares` library targets. The libpcap package for vcpkg doesn't do that though, so we still have to handle it like a bare library/include path.